<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoConnect</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="contactList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>25</number>
        </attribute>
        <column>
         <property name="text">
          <string>Имя</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>IP-адрес</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="deleteBtn">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addContactBtn">
          <property name="text">
           <string>Добавить контакт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="callBtn">
          <property name="text">
           <string>Позвонить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/image/image/logo-eleron.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/image/image/logo-rosatom.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>210</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Настройки Аудио</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Устройство воспроизведения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="outputDevicesBox">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="microBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Микрофон</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="inputDevicesBox">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Громкость записи</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="volumeMicSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Настройки Видео</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string>Видеокамера</string>
           </property>
           <property name="shortcut">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cameraDevicesBox">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Качество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="qualitySlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>80</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
           <property name="sliderPosition">
            <number>20</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Кадров в секунду</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fpsSpinBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="uloadLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	background-color : black;
	color: green;
	font: 75 12pt &quot;Arial&quot;;
}</string>
        </property>
        <property name="text">
         <string>0 kbps</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dloadLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	background-color : black;
	color: red;
	font: 87 12pt &quot;Arial&quot;;
}</string>
        </property>
        <property name="text">
         <string>0 kbps</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="soundSettings">
   <property name="text">
    <string>Настройки звука</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="connectionSetings">
   <property name="text">
    <string>Настройки соединения</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
